decidim-results/app/views/decidim/results/results/show.html.erb

```diff
@@ -0,0 +1,48 @@
+<% content_for(:title, translated_attribute(result.title)) %>
+
+<div class="row column view-header">
+  <h2 class="heading2"><%= translated_attribute result.title %></h2>
+</div>
+<div class="row">
+  <div class="columns section view-side mediumlarge-4 mediumlarge-push-8
+              large-3 large-push-9">
+    <div class="card extra">
+      <div class="definition-data">
+        <div class="definition-data__item definition-data__item--double">
+          <span class="definition-data__title">Suports</span>
+					<span class="definition-data__number">27</span>
+        </div>
+        <div class="definition-data__item definition-data__item--double">
+          <span class="definition-data__title">Propostes</span>
+					<span class="definition-data__number">130</span>
+        </div>
+        <div class="definition-data__item definition-data__item--double">
+          <span class="definition-data__title">Comentaris</span>
+					<span class="definition-data__number">2567</span>
+        </div>
+        <div class="definition-data__item definition-data__item--double">
+          <span class="definition-data__title">Participacions</span>
+					<span class="definition-data__number">130</span>
+        </div>
+        <div class="definition-data__item definition-data__item--double">
+          <span class="definition-data__title">Intervencions</span>
```

Comment:
In decidim.barcelona I created a service to compute these values: https://github.com/AjuntamentdeBarcelona/decidim.barcelona/blob/master/app/services/action_plan_statistics_calculator.rb

I think the "intervencions" wasn't used and the "participacions" is the sum of votes of all result's proposals.